smurfs/src/actions/index.js

```diff
@@ -13,3 +15,57 @@
    U - updateSmurf
    D - deleteSmurf
 */
+
```

Comment:
Its good practice and very common in the industry to use constants instead of strings when you need to use the same string in places. You may want to start using ACTION_TYPES as constants in place of strings